pkl-codegen-java/src/test/kotlin/org/pkl/codegen/java/JavaCodeGeneratorTest.kt

```diff
@@ -263,8 +263,8 @@ class JavaCodeGeneratorTest {
         |      return new ClassWithDeprecatedProperty(deprecatedProperty);
         |    }
       """
-        .trimMargin()
-    assertThat(javaCode).contains(expectedPropertyDef).contains(expectedWithMethodDef)
+    assertContains(expectedPropertyDef, javaCode)
+    assertContains(expectedWithMethodDef, javaCode)
```

Comment:
I actually don't like our `assertFoo` methods much; it reads entirely different than the (quite nice) `assertj` style. Not blocking (happy to follow up myself), but might I suggest replacing `assertContains` with
```
  private fun <T : AbstractStringAssert<T>> AbstractStringAssert<T>.containsTrimmed(part: String): AbstractStringAssert<T> {
    val trimmed = part.trim().trimMargin()
    if (!part.contains(trimmed)) {
      isEqualTo(trimmed)
    }
    return this
  }
```

and then
```suggestion
    assertThat(javaCode)
      .containsTrimmed(expectedPropertyDef)
      .containsTrimmed(expectedWithMethodDef)
```

[throughout]

Reply:
working on this (will take some time)